styles/about.css

```diff
@@ -0,0 +1,71 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+} 
+
+.home {
+  font-weight: bold;
+}
+
+header {
+  background-color: #f9d199;
+  max-width: 100%;
+  height: 84px; 
+}
+
+.logo {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  margin-top: 30px;
+  margin-left: 70px; 
+}
+
+.menu-header {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.menu-header ul {
+  list-style: none;
+}
+
+.menu-header ul li {
+  display: inline;
+}
+
+ul {
```

Comment:
@Weslley-Mistura , avoid applying global styles for a tag. I suggest you to do something as the line 46 of the code.